docs/index.md

```diff
@@ -53,6 +53,11 @@ uses Scala {{site.SCALA_BINARY_VERSION}}. You will need to use a compatible Scal
 For Python 3.9, Arrow optimization and pandas UDFs might not work due to the supported Python versions in Apache Arrow. Please refer to the latest [Python Compatibility](https://arrow.apache.org/docs/python/install.html#python-compatibility) page.
 For Java 11, `-Dio.netty.tryReflectionSetAccessible=true` is required additionally for Apache Arrow library. This prevents `java.lang.UnsupportedOperationException: sun.misc.Unsafe or java.nio.DirectByteBuffer.(long, int) not available` when Apache Arrow uses Netty internally.
 
+```python
+# PySpark
+SparkSession.builder.config('spark.driver.extraJavaOptions', '-Dio.netty.tryReflectionSetAccessible=true').getOrCreate()
```

Comment:
@Nozziel, I think we should better just mention both configurations `spark.driver.extraJavaOptions` and `spark.executor.extraJavaOptions` because there are many other ways to set the configurations, for example, Spark submit script, Spark configuration file, etc. In addition, Arrow exposes a set of developer APIs that are not specific to PySpark side.

Also, it would be great if we file a JIRA, and fix the PR title accordingly. See also http://spark.apache.org/contributing.html

Reply:
updated with jira link. 
The pyspark example is intended for those that rarely ever change the config...
Those that would not think of using 'spark.driver.extraJavaOptions' in the first place... 
And should be clear enough now for everybody to adapt it to their prefered way to set this config setting.

And hopeful this underlying issue gets resolved soon making this all moot. But till then I hope to save a few people a bit of time.

Reply:
I would prefer to just explain the configuration keys alone given that there are multiple ways of setting it. Also, I think we should set `spark.executor.extraJavaOptions`. I suspect setting `spark.driver.extraJavaOptions` will only work with toPandas and createDataFrame with Arrow optimization (?).